packages/dashboard/src/components/sidePanel/sections/propertiesAlarmSection/index.tsx

```diff
@@ -0,0 +1,69 @@
+import React, { FC } from 'react';
```

Comment:
This file was `propertiesAlarmSection.tsx`. I moved it here with a new component `PropertyComponent` and renamed it.